modules/ROOT/pages/tools/neo4j-admin/neo4j-admin-import.adoc

```diff
@@ -85,6 +88,7 @@ neo4j-admin database import full [-h] [--expand-commands] [--verbose] [--auto-sk
                                  [--bad-tolerance=<num>] [--delimiter=<char>] [--format=<format>]
                                  [--high-parallel-io=on|off|auto] [--id-type=string|integer|actual]
                                  [--input-encoding=<character-set>] [--max-off-heap-memory=<size>] [--quote=<char>]
+                                 [--schema=<path>]
                                  [--read-buffer-size=<size>] [--report-file=<path>] [--threads=<num>] --nodes=[<label>[:
                                  <label>]...=]<files>... [--nodes=[<label>[:<label>]...=]<files>...]...
                                  [--relationships=[<type>=]<files>...]... <database>
```

Comment:
from the build failure does `--schema` need to be included somewhere here too?

Reply:
yes, `--schema` has to be included. But I don't understand where to put it to make our test happy.
Also, if I read the test's results correctly,  the `--schema` option must be in the incremental part too.

Reply:
It's a weird one - it needs to be in the basic command so it to can be picked up by all 3 import types (full, incremental and SPDY) - even in community runs
Order wise, I think they're alphabetical so after --relationships?